test/src/client/entity-editor/validators/test-edition.js

```diff
@@ -441,37 +441,37 @@ function describeValidateForm() {
 		expect(result).to.be.true;
 	});
 
-	it('should reject an Object with an invalid alias editor', () => {
+	it('should pass an Object with an empty author credit editor', () => {
 		const result = validateForm(
 			{
 				...validForm,
-				aliasEditor: INVALID_ALIASES
+				authorCreditEditor: {}
 			},
 			IDENTIFIER_TYPES
 		);
-		expect(result).to.be.false;
+		expect(result).to.be.true;
```

Comment:
Following my comment above about a specific boolean for optional ACs, we want to be able to test separately that:
- empty AC with `authorCreditDisabled = false` should fail
- empty AC with `authorCreditDisabled = true` should pass
- valid AC with `authorCreditDisabled = true` … ? What should we do here? Should validation fail? Or should we accept it?

Reply:
No, since AC field should be disable at that point. passing test will cause unnecessary confusion.